packages/api-db-mongodb/src/migration.ts

```diff
@@ -628,8 +628,13 @@ export const Migrations: Migration[] = [
         const paymentProvidersCustomersArray: PaymentProviderCustomer[] = []
         const paymentProviderCustomers = Object.keys(user.paymentProviderCustomers)
         paymentProviderCustomers.forEach(ppc => {
-          // @ts-ignore
-          const userPPC = user.paymentProviderCustomers[ppc]
+          interface OldPaymentProviderCustomer extends PaymentProviderCustomer {
```

Comment:
Replicating the old interface, as `id` does not exist anymore on `PaymentProviderCustomer` but `customerID` now does.